### DISCUSSION TITLE
Link to products in FluxBuilder menus

### DISCUSSION THREAD

**Discussion by gregkeys** [inspireui/support#5495] [Working on Features]
We are trying to add a link to a product on the side menu and in the profile settings view. The product is for donations to support our project. 

We are only able to add url's, page ideas or images, using firebase dynamic links we are able to get to a product in the app but the experience is not smooth, it opens a browser, asks the user how to open the link if they the app it reloads the app in order to get to the page.

It would be much better if clicking the link in the side menu could simply open a product page making a much better experience. we are able to accomplish this with banners in the main screen but we would prefer a more obscure link in the side menu for our donation/support link.

Please add the ability to link to products and categories for the items in the sidebar, topbar, footer and settings page

**Comment by tjain90** [comment]
Yes that will make our app more robust.

**Comment by simonpham** [comment]
This feature is supported already. Please download our latest version: [https://github.com/inspireui/fluxbuilder/releases
](https://github.com/inspireui/fluxbuilder/releases)
